### PR TITLE
feat(RHINENG-13842): Change rule tree icon

### PR DESCRIPTION
**Pull request by LightOfHeaven1994**
I double checked it with UXD and new icon looks good.
image (33)

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: src/Frameworks/AsyncTableTools/components/TreeIcon.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const TreeIcon = () => (
+  <img
+    src={require('../../../../static/images/pf-icon-tree-view.svg')}
```
Comment (bastilian) [inline src/Frameworks/AsyncTableTools/components/TreeIcon.js:5]: Is it not in PF icons?

File: src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { ListIcon } from '@patternfly/react-icons';
-import Bars from '../../../components/Bars';
+import TreeIcon from '../../../components/TreeIcon';
```
Comment (johnsonm325) [inline src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js:3]: Agree with @bastilian, Replace the TreeIcon file and use `import { TreeIcon } from '@patternfly/react-icons';`.

**Review by johnsonm325 — CHANGES_REQUESTED** [review]
Remove those TreeIcon files, the svg file, and just import TreeIcon from patternfly icons.

File: src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js (hunk 2 above)
Comment (LightOfHeaven1994) [reply src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js:3]: @johnsonm325 @bastilian I wasn't able to find tree icon [here](https://www.patternfly.org/design-foundations/icons/) . I tried to paste the thing you suggested and see Christmas tree :smile: 
image

Comment (LightOfHeaven1994) [reply src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js:3]: I think that's the reason why UXD had this icon saved in some google shared folder as there are no such icon on pf side

File: src/Frameworks/AsyncTableTools/components/TreeIcon.js (hunk 1 above)
Comment (LightOfHeaven1994) [reply src/Frameworks/AsyncTableTools/components/TreeIcon.js]: found it, had to take a look into code and found a link to all available icons https://react-staging.patternfly.org/icons/

File: src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js (hunk 2 above)
Comment (LightOfHeaven1994) [reply src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js]: @johnsonm325 good catch, found correct icon here found it, had to take a look into code and found a link to all available icons https://react-staging.patternfly.org/icons/

Comment (mkholjuraev) [reply src/Frameworks/AsyncTableTools/hooks/useTableView/views/index.js]: Christmas is coming :)

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**